drivers/homeless_summary_report/app/models/homeless_summary_report/report.rb

```diff
@@ -823,70 +823,19 @@ def self.available_variants
     end
 
     def self.demographic_variants
```

Comment:
What do you think about moving this to the `HudUtility2024` class? Then it could feed the various filter scopes, and other bits, and can be expanded with new keys as necessary.

Reply:
Looks good, thank you.